utils/lineinfile/lineinfile.go

```diff
@@ -47,16 +47,23 @@ func NewAbsentParams() Params {
 }
 
 func (l LineInFile) Present(params Params) error {
```

Comment:
Some docs would be nice what this function is doing (I guess check if a particular parameter is present).

Reply:
It was already added in #207 so it's a bit unrelated to this PR but I can add some docs later. As explained in the other PR, it's akin to Ansible's lineinfile module, where "present" ensures that a line in file exists, "absent" ensures it's not.